src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Wasm/MyExtensionsApp._1.Wasm.csproj

```diff
@@ -9,6 +9,7 @@
 		<WasmPWAManifestFile>manifest.webmanifest</WasmPWAManifestFile>
 		<!--#endif -->
 		<AppDesignerFolder>Properties</AppDesignerFolder>
+		<WasmShellWebAppBasePath>/</WasmShellWebAppBasePath>
```

Comment:
We may need to link to the docs here, to explain what the impact is. https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-deep-linking.html

Reply:
Actually, let's use https://aka.platform.uno/wasm-deeplink